tests/integ/test_cte.py

```diff
@@ -902,10 +902,6 @@ def test_window_function(session):
 
 
 def test_in_with_subquery_multiple_query(session):
-    if session._sql_simplifier_enabled:
-        pytest.skip(
```

Comment:
can you add a test in the regular test_dataframe to check the query generated without any new optimizations? and also check the schema query by calling df.schema

Reply:
added a test but not sure what exactly to test with schema.

should I do something like
```
df_filter._plan.queries[-1] not in df_in._plan.schema_query
```